cli/cmd/create/create.go

```diff
@@ -56,7 +56,7 @@ type Create struct {
 	HealthInterval         string            `desc:"Time between running the check (ms|s|m|h)" default:"0s"`
 	HealthSuccessThreshold int               `desc:"Consecutive successes needed to report healthy"`
 	HealthTimeout          string            `desc:"Maximum time to allow one check to run (ms|s|m|h)" default:"0s"`
-	HealthURL              string            `desc:"URL to hit to check health (example: http://localhost:8080/ping)"`
+	HealthURL              string            `desc:"URL to hit to check health (example: http://:8080/ping)"`
```

Comment:
why this change?

Reply:
It's misleading to include in example, localhost won't work for most cases:
```The kubelet sends the probe to the pod’s IP address, unless the address is overridden by the optional host field in httpGet. If scheme field is set to HTTPS, the kubelet sends an HTTPS request skipping the certificate verification. In most scenarios, you do not want to set the host field. ```